rdflib/extras/infixowl.py

```diff
@@ -383,17 +387,71 @@ def __init__(self, identifier=None, graph=None):
             except Exception:  # pragma: no cover
                 pass  # pragma: no cover
 
+    def snc(self, graph, bnc=False):
```

Comment:
```suggestion
    def snc(self, graph):
```

As far as I can tell, the `bnc` argument is not being used, can I remove it?

Also, a bit of a tangent, what is `snc` an abbreviation for?

Reply:
I think a more complete name for this may be a better option, like "subject_node_closure" - makes for more verbose code but also maybe slightly easier for others to understand.